shopfloor_dangerous_goods/models/product_product.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    is_lq_product = fields.Boolean(compute="_compute_is_lq_product")
```

Comment:
I don't think this should stay here. All these fields should stay in another module.